resources/vue/src/components/account/messages/MessageListItem.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <tr>
+        <th scope="row">
+            <a :href="'/profile/message?opponent=' + message.opponent.id" class="list-group-item list-group-item-action">{{ message.subject }}</a>
```

Comment:
All links if they are internal should use <router-link>, otherwise they'll force-refresh your page and most of the SPA routing benefits will be gone.